WhaleSpotting/Models/Response/SpeciesResponse.cs

```diff
@@ -0,0 +1,29 @@
+
+using WhaleSpotting.Models.Database;
+
+namespace WhaleSpotting.Models.Response;
+
+public class SpeciesResponse
+{
+    public int Id { get; set; }
+    public string ImageUrl { get; set; }
+    public string Name { get; set; }
+    public TailType TailType { get; set; }
+    public TeethType TeethType { get; set; }
+    public WhaleSize Size { get; set; }
+    public string Colour { get; set; }
+    public string Location { get; set; }
+    public string Diet { get; set; }
+    public SpeciesResponse (WhaleSpecies species)
```

Comment:
Please can we add an extra line above the constructor to make this easier to read?